chromadb/db/impl/grpc/client.py

```diff
@@ -247,7 +247,10 @@ def get_collections(
         name: Optional[str] = None,
         tenant: str = DEFAULT_TENANT,
         database: str = DEFAULT_DATABASE,
+        limit: Optional[int] = None,
+        offset: Optional[int] = None,
     ) -> Sequence[Collection]:
+        # TODO: implement limit and offset in the gRPC service
```

Comment:
coordinate with @Ishiihara about this

Reply:
Yeah lets stack a PR so we can maintain functionality parity

Reply:
Yes. Sounds good!